pkg/grafana/client/client.go

```diff
@@ -9,19 +9,18 @@ import (
 
 var grafanaURL *url.URL
 
-func init() {
+const (
+	clientConfigNumRetries = 3
+)
+
+func GenerateGrafanaClient(grafanaURLstring string, adminUserCredentials AdminCredentials, tlsConfig TLSConfig) (*grafana.GrafanaHTTPAPI, error) {
 	var err error
-	grafanaURL, err = url.Parse("http://grafana.monitoring.svc.cluster.local")
+
+	grafanaURL, err = url.Parse(grafanaURLstring)
```

Comment:
Why not parse it in main.go and have an url in config?

Reply:
I updated my branch to parse it in main.go @QuentinBisson , ready for re-review :wink: